includes/class-mailer.php

```diff
@@ -349,8 +349,13 @@ public static function mention( $activity, $user_ids ) {
 			}
 
 			$actor_id = $actor->get_id();
-			if ( \in_array( $actor_id, (array) $activity['cc'], true ) ) {
-				$recipients[ $user_id ] = $actor_id;
+			if ( \in_array( $actor_id, (array) $activity['cc'], true ) && isset( $activity['object']['tag'] ) ) {
+				foreach ( (array) $activity['object']['tag'] as $tag ) {
+					if ( object_to_uri( $tag ) === $actor_id ) {
+						$recipients[ $user_id ] = $actor_id;
+						break;
+					}
+				}
 			}
```

Comment:
why even check the `cc` then? I would simply go only through the mentions instead.

Reply:
Sure!